c7n/resources/glue.py

```diff
@@ -205,3 +206,88 @@ def process(self, resources):
                 client.delete_crawler(Name=r['Name'])
             except client.exceptions.EntityNotFoundException:
                 continue
+
+
+@resources.register('glue-database')
+class GlueDatabase(QueryResourceManager):
+
+    class resource_type(object):
```

Comment:
resource_type is a subclass of TypeInfo now, all the None fields can be removed.